src/services/fee-services.ts

```diff
@@ -1,7 +1,6 @@
 import Axios from 'axios'
-import { Fee, FilingData, PayData } from '@/interfaces'
-import ConfigHelper from '@/utils/config-helper'
-import { SessionStorageKeys } from '@/enums/sbc-common-components-constants'
+import { Fee, FilingData, PayData } from '@bcrs-shared-components/interfaces'
+import { SessionStorageKeys } from '@bcrs-shared-components/enums/sbc-common-components-constants'
```

Comment:
I _think_ this is correct, which makes me wonder how this worked previously.

What we want is:
- these imports work when we run this code in THIS project (eg, via unit tests or Storybook)
- these imports work when this component is imported into another project (ie, that project will have the right dependencies)

Please test these changes. If you cannot, I recommend not making these changes.

Reply:
I wanted to avoid unnessaary changes but `fee-service` raises an error when I install bcrs-common/services in another project.

Reply:
What error, by the way?

Reply:
Makes sense. The files can't be found because they're not in your local project.

I'll resolve this conversation and help you if you find anything else along the way. Looks good so far.